Framework/Algorithms/src/CalculateDynamicRange.cpp

```diff
@@ -80,8 +80,12 @@ void CalculateDynamicRange::calculateQMinMax(MatrixWorkspace_sptr workspace,
   const auto &spectrumInfo = workspace->spectrumInfo();
   double min = std::numeric_limits<double>::max(),
          max = std::numeric_limits<double>::min();
+  // PARALLEL_FOR_NO_WSP_CHECK does not work with range-based for so NOLINT this
+  // block
   PARALLEL_FOR_NO_WSP_CHECK()
-  for (int64_t index = 0; index < static_cast<int64_t>(indices.size());
+  for (int64_t index = 0;
+       index <
+       static_cast<int64_t>(indices.size()); // NOLINT (modernize-for-loop)
```

Comment:
Can `NOLINT` be moved to the end of the `for` expression (next line down)?

Reply:
Good spot, not sure why I put it in the middle of the for